src/Servers.jl

```diff
@@ -382,14 +386,16 @@ Otherwise, execute stream processing function `f`.
 """
 function handle_transaction(f::Function, t::Transaction;
                             final_transaction::Bool=false,
-                            verbose::Bool=false, kw...)
+                            verbose::Bool=false,
+                            conn::Int=1, kw...)
 
     request = HTTP.Request()
     http = Streams.Stream(request, t)
 
     try
         startread(http)
     catch e
+        @show dump(e)
```

Comment:
Doing console IO before `rethrow` will currently corrupt the backtrace because of https://github.com/JuliaLang/julia/issues/19979#issuecomment-344453118
Maybe better to only `@show` if `DEBUG_LEVEL` is set? Or leave it there but commented out like the two below it?